### PR TITLE
Adding auth to grpc ng

### PR DESCRIPTION
**Pull request by daliclovr**
Added a GRPC interceptor to NodeGuard client to send the proper auth header. 

https://www.notion.so/clovrlabs/Phase-3-Live-4a570b5e538b4a0aa4b6980007d68977?p=80f65dac3561479aad3e7c10158d4eaf&pm=c

Regarding the value of new environment variable `NODEGUARD_API_KEY` i will automatically create this data for development purposes in NodeGuard database :). Please, wait for this changes to come!

I introduced a couple of changes as well:
- Used Option pattern to encapsulate the `getConn` method. We have several grpc options which are common to all of our clients but only NG needed auth so far. I used this pattern to make it optional and customizable per client, keeping the base options for everyone.
- Created `.env` file :) should be the agnostic way to go! Please, `vscode` users update your configs to add new variable.
- Improved some error messages... and move one error regarding connection to NG. I noticed that placing after checking if liquidation rules are empty masked connection errors.

### CODE REVIEW

File: rpc/rpc.go [hunk 1]
```diff
@@ -18,6 +19,34 @@ import (
...
+		grpcEndpoint,
+		options...)
+
+	if err != nil {
+		log.Errorf("did not connect: %v", err)
+		return nil, err
+	}
+
+	return conn, nil
+}
```
Comment (markettes) [inline rpc/rpc.go:44]: Liked this a lot. I copied xD

**Review by markettes — APPROVED** [review]
LGTM

File: .github/pull_requeste_template.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+## Notion task
+
+Paste your notion task link here.
+
+
+Take a look into our [guidelines](https://www.notion.so/clovrlabs/Guidelines-3e2dcb1f8be14d469a701f997a4ff1a0).
+This repo must contain:
+
+- `Justfile`
+- A single `docker-compose` file. In case of fork, locate our own under a `docker-elenpay` folder
```
Comment (Jossec101) [inline .github/pull_requeste_template.md:14]: This link is private, also check the name of the markdown.

**Review by Jossec101 — APPROVED** [review]

File: .github/pull_requeste_template.md (hunk 2 above)
Comment (daliclovr) [reply .github/pull_requeste_template.md:14]: Good catch 😄

**Review by gcaracuel — APPROVED** [review]

**Pull request merged**